utils/polus-filepattern-util/examples/Montage.py

```diff
@@ -0,0 +1,12 @@
+import bfio, filepattern, pathlib, pprint
+
+filepath = pathlib.Path(__file__).parent
+filepath = filepath.joinpath('data/Small_Fluorescent_Test_Dataset/image-tiles/')
+
+fp = filepattern.FilePattern(filepath,'img_r00{y}_c00{x}.tif')
+
+pprint.pprint(fp.get_matching(X=1))
```

Comment:
This is a convention I came up with when working on `bfio`. Basically, input variables that can be either a single value or list of values I capitalize to signify that it could be one or more values, while lowercase means it will definitely only be an integer. It is case sensitive, but it doesn't need to be. It helped me keep things organized when I was writing it, but I could change it if you would think it would help with clarity and ease of use.